com.unity.render-pipelines.universal/Shaders/Particles/ParticlesLit.shader

```diff
@@ -120,7 +120,7 @@ Shader "Universal Render Pipeline/Particles/Lit"
             #include "Packages/com.unity.render-pipelines.universal/Shaders/Particles/ParticlesLitForwardPass.hlsl"
             ENDHLSL
         }
-	// ------------------------------------------------------------------
+	    // ------------------------------------------------------------------
```

Comment:
It looks like it is missing space and probably tab is here, a bit conffused about github here :D